tasks/stratum1.yml

```diff
@@ -8,6 +8,11 @@
 - name: Include key setup tasks
   ansible.builtin.include_tasks: keys.yml
 
+- name: Enable the Apache2 module wsgi
+  community.general.apache2_module:
```

Comment:
In https://docs.ansible.com/ansible/latest/collections/community/general/apache2_module_module.html

> This does not work on RedHat-based distributions.

You may want to restrict this to Debian family OS.

Reply:
Looks like `init_debian.yml` might be the file to edit.